static/app/views/dashboardsV2/widgetBuilder/yAxisSelector.tsx

```diff
@@ -183,79 +191,56 @@ function YAxisSelector({
-          filterAggregateParameters={filterAggregateParameters(fieldValue)}
-        />
-      </QueryFieldWrapper>
-    );
-  } else {
-    fieldContents = (
-      <React.Fragment>
-        {fields.map((fieldValue, i) => (
-          <QueryFieldWrapper key={`${fieldValue}:${i}`}>
-            <QueryField
-              fieldValue={fieldValue}
-              fieldOptions={fieldOptions}
-              onChange={value => handleChangeField(value, i)}
-              filterPrimaryOptions={filterPrimaryOptions}
-              filterAggregateParameters={filterAggregateParameters(fieldValue)}
-              otherColumns={fields}
-            />
-            {(canDelete || fieldValue.kind === 'equation') && (
-              <DeleteButton onDelete={event => handleRemove(event, i)} />
-            )}
-          </QueryFieldWrapper>
-        ))}
-        {!hideAddYAxisButtons && (
-          <Actions>
-            <AddButton title={t('Add Overlay')} onAdd={handleAdd} />
-            <AddButton title={t('Add an Equation')} onAdd={handleAddEquation} />
-          </Actions>
-        )}
-      </React.Fragment>
-    );
-  }
-
   return (
-    <Field
-      data-test-id="y-axis"
-      label={t('Y-Axis')}
-      inline={false}
-      style={{padding: `${space(2)} 0 24px 0`, ...(style ?? {})}}
-      flexibleControlStateSize
-      error={errors?.fields}
-      required
-      stacked
-    >
-      {fieldContents}
+    <Field inline={false} flexibleControlStateSize error={fieldError} required stacked>
+      {fields.map((fieldValue, i) => (
+        <QueryFieldWrapper key={`${fieldValue}:${i}`}>
+          <QueryField
+            fieldValue={fieldValue}
+            fieldOptions={fieldOptions}
+            onChange={value => handleChangeField(value, i)}
+            filterPrimaryOptions={filterPrimaryOptions}
+            filterAggregateParameters={filterAggregateParameters(fieldValue)}
+            otherColumns={fields}
+          />
+          {(canDelete || fieldValue.kind === 'equation') && (
+            <Button
+              size="zero"
+              borderless
+              onClick={event => handleRemove(event, i)}
+              icon={<IconDelete />}
+              title={t('Remove this Y-Axis')}
+              aria-label={t('Remove this Y-Axis')}
+            />
+          )}
+        </QueryFieldWrapper>
+      ))}
+      {!hideAddYAxisButtons && (
+        <Actions gap={1}>
+          <AddButton title={t('Add Overlay')} onAdd={handleAdd} />
+          <AddButton title={t('Add an Equation')} onAdd={handleAddEquation} />
+        </Actions>
+      )}
     </Field>
   );
 }
 
 export default YAxisSelector;
 
-const Actions = styled('div')`
-  & button {
-    margin-right: ${space(1)};
-  }
-`;
-
 const QueryFieldWrapper = styled('div')`
   display: flex;
   align-items: center;
   justify-content: space-between;
-  margin-bottom: ${space(1)};
+
+  :not(:last-child) {
+    margin-bottom: ${space(1)};
```

Comment:
![image](https://user-images.githubusercontent.com/29228205/154967857-681a7fe8-f163-4dbd-a6a7-4493e76c8f0c.png)

the last item does not need to have a margin-bottom